content/blog/3-same-subject-continued-concerning-dangers/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: The Same Subject Continued: Concerning Dangers from Foreign Force and Influence
+date: "2020-11-18"
+originalAuthor: John Jay
+description: "It is better to have a single, united government when it comes to the dangers of war an international relations."
+---
+
+It hardly takes a genius to recognize that the citizens of any country (especially intelligent and well-informed citizens like Americans) rarely adopt incorrect opinions about what is good for them--at least not for very long. So, the fact that our countrymen have long been united in believing that we should stay united under a single, sufficiently powerful federal government means that it's probably a good idea.
+
+Indeed, the more I look into the reasons for this opinion, the more convinced I am that they are compelling and decisive.
+
+There are lots of different things that members of a democracy need to pay attention to, but providing for their collective safety is probably the first. "Safety" means lots of things to different people and in different situations, so it's important to define what I mean precisely.
+
+For this essay, I am only considering safety as it relates to maintaining peace from foreign violence and domestic violence. I'll start with foreign violence. So, let's think about whether people are right that a national government really affords the best security against hostilities from abroad.
+
+The number of wars in the world is proportionate to the number and weight of real or pretended causes. If this is right, then we should think about whether a united America or a divided America is likely to produce more just causes for war; if it turns out that a united America will give fewer causes for war, then a union will be more likely to lead to peace with other nations.
```

Comment:
This is good, but still a bit hard to understand. I like the paraphrasing in the next paragraph, in that it simplifies what is actually being said. I seem to remember this paragraph basically saying, "It's more likely to get into a fight with at least one of 13 colonies than it is to get into a fight with just one united nation." I could be wrong, and I'm not looking at it right now.